### PR TITLE
feat: add permissions.

### PR DESCRIPTION
**Pull request by marvinIsSacul**

### CODE REVIEW

**Comment by rcrichton** [comment]
Task linked: [CU-86c0bkuhz Conditionally show the menu items based on the permissions the user has](https://app.clickup.com/t/86c0bkuhz)

**Review by drizzentic — CHANGES_REQUESTED** [review]

File: packages/header-app/src/types/index.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  'contact-list-view': boolean
+  'contact-list-manage': boolean
+  'certificates-view': boolean
+  'certificates-manage': boolean
+  'logs-view': boolean
+  'import-export': boolean
+  'auth-type-view-all': boolean
+  'events-view-all': boolean
+  'metadata-view-all': boolean
+  'metadata-manage-all': boolean
```
[the remark this answers is not in the document]
Comment (marvinIsSacul) [reply packages/header-app/src/types/index.ts]: Done.

File: packages/header-app/src/components/openhim.appbar.component.tsx [hunk 2]
```diff
@@ -107,91 +183,41 @@ export default function OpenhimAppBar() {
...
-      />
-  )
-
   const fetchMe = async () => {
     if (window.location.href.includes('#!/forgot-password')) return
```
Comment (drizzentic) [inline packages/header-app/src/components/openhim.appbar.component.tsx:188]:
```suggestion
    if (window.location.href.includes('#!/login')) return
```

**Review by drizzentic — APPROVED** [review]
LGTM!

**Pull request merged**